metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi-lexical-grammar.rule

```diff
@@ -432,17 +434,7 @@ optional_parameter: "optional" WS_INLINE parameter
 
 each_expression:    "each" WS_INLINE each_expression_body
 
-// relational_expression rule is different then relational expression used in below grammar to resolve "each" construct
 each_expression_body:   function_body
```

Comment:
in simple terms - why are we able to remove this?

Reply:
the addition to unary operator makes this one work correctly